stratisd_cert.py

```diff
@@ -334,6 +334,17 @@ def _test_permissions(self, dbus_method, args, permissions, *, kwargs=None):
 
         self.assertEqual(_permissions_flag, permissions)
 
+    def test_access_stratisd_man_page(self):
```

Comment:
You should go ahead and make a separate class for this test. Call the class something like StratisdManPageCertify and make sure it inherits from StratisCertify.